package.json

```diff
@@ -5,15 +5,20 @@
   "repository": "https://github.com/choonkending/react-webpack-node",
   "main": "index.js",
   "scripts": {
-    "start": "node server/index.js",
-    "build": "webpack --progress",
-    "watch": "webpack --watch --progress",
+    "start": "NODE_ENV=production node server/index.js",
+    "dev": "NODE_ENV=development node server/index.js",
+    "build": "webpack --progress --colors --config ./webpack/webpack.config.js",
+    "buildDev": "webpack --progress --colors --config ./webpack/webpack-hot-dev-server.config.js",
```

Comment:
`npm run dev` should recompile and change your page on file save, without a `--watch`.